rebar.config.script

```diff
@@ -1,19 +1,9 @@
 %% -*- erlang -*-
-code:is_loaded(rebar3) =:= false andalso code:load_file(rebar3),
-CoverallsPlugin = case erlang:function_exported(rebar3, version, 0) of
-    true -> {coveralls, {git, "git://github.com/markusn/coveralls-erl.git", {branch, "master"}}};
-    false -> rebar_coveralls
-end,
 case os:getenv("TRAVIS") of
     "true" ->
         JobId   = os:getenv("TRAVIS_JOB_ID"),
         CONFIG1 = lists:keystore(coveralls_service_job_id, 1, CONFIG, {coveralls_service_job_id, JobId}),
-        {value, {plugins, Plugins}} = lists:keysearch(plugins, 1, CONFIG1),
-        lists:keystore(plugins, 1, CONFIG1, {plugins, Plugins ++ [CoverallsPlugin]})
```

Comment:
This change the rebar config logic. With the old style the plugin was only included when running on travis, with the new logic it is always included.

Have you checked that this does work when running a `rebar3 ct` manually? It should invoke coverall in that case....

Reply:
It won't invoke coverall because `rebar.config` contains these lines:

```
{do_coveralls_after_ct, false}.
{do_coveralls_after_eunit, false}.
```

I think that ok if we have coveralls plugin permanently added in config. And yes, `rebar3 ct` works locally.